README.rst

```diff
@@ -634,6 +634,11 @@ Choose from the list of available fixers:
                         closing semicolon are
                         prohibited.
 
+* **multiple_unset** [contrib]
```

Comment:
they don't want it at SF level, so if this fixer ever makes it into the project it should be at contrib level